lua/conform/formatters/liquidsoap-prettier.lua

```diff
@@ -0,0 +1,10 @@
+local util = require("conform.util")
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/savonet/liquidsoap-prettier",
+    description = "A binary to format Liquidsoap scripts",
+  },
+  command = util.from_node_modules("liquidsoap-prettier"),
+  args = { "$FILENAME" },
```

Comment:
This command is writing the formatting to stdout, but it's reading it from the file. I think you need to set
```lua
args = { "-w", "$FILENAME" },
stdin = false,
```

Reply:
Both work but that way I cleaner ye